GitUI/GitUIExtensions.cs

```diff
@@ -271,5 +272,27 @@ public static Control FindFocusedControl(this ContainerControl container)
                 }
             }
         }
+
+        public static string GetGitVersionString()
+        {
+            if (!File.Exists(AppSettings.GitCommandValue))
```

Comment:
Arguably this is unreachable code - if git is not installed the app will crash, see #5562